src/kernel/src/memory/page.rs

```diff
@@ -12,6 +12,14 @@ impl<'a> VPages<'a> {
         Self { mm, ptr, len }
     }
 
+    pub fn add(&self, offset: usize) -> Self {
+        Self {
+            mm: self.mm,
+            ptr: unsafe { self.ptr.add(offset) },
+            len: self.len - offset,
+        }
+    }
+
```

Comment:
This will cause `munmap` multiple times on the same region, which is a bug.

Reply:
What would you say would be the best method to fixing this?